src/parameter_declarations.ts

```diff
@@ -192,7 +192,8 @@ Parameter.declare({
 	name: AllParameter,
 	aliases: ['--all'],
 	description: `(OPTIONAL) gets all contents within this folder, including child files/folders`,
-	type: 'boolean'
+	type: 'boolean',
+	forbiddenConjunctionParameters: [MaxDepthParameter]
```

Comment:
Does it make sense to also list the --all parameter as forbidden in conjunction with the all parameter? Is it functionally different to mark them forbidden in one direction or the other?